### ISSUE TITLE
Installation not possible

### ISSUE THREAD

**Issue by MyneLife** [directus/directus#2439]
# Bug Report
I'd like to try out Directus but I'm unable to get Directus up and running.

## Steps to Reproduce
Follow the [official installation instructions](https://docs.directus.io/getting-started/installation.html)
1. Clone the git repo to `/var/www/XXX/directus` using `git clone https://github.com/directus/directus.git` (commit `e49e5d28f4c4506f18567db613fb0d471d8df62c`)
2. Apply required Apache settings (mod_rewrite, AllowOverride, redirect to `/public`)
3. Visit `/admin`

## Expected Behavior
The configuration screen should show up.

## Actual Behavior
Redirection to login screen without possibilty to start the configuration process.

Error in Apache log:
```
[Fri Sep 27 14:24:32.891821 2019] [php7:error] [pid 23581] [client XXX] PHP Fatal error:  Uncaught Directus\\Logger\\Exception\\InvalidLoggerConfigurationException: Invalid Configuration for logger "path" variable. in var/www/XXX/directus/src/core/Directus/Application/CoreServicesProvider.php:142
Stack trace:
#0 /var/www/XXX/directus/vendor/pimple/pimple/src/Pimple/Container.php(118): Directus\\Application\\CoreServicesProvider->Directus\\Application\\{closure}(Object(Directus\\Application\\Container))
#1 /var/www/XXX/directus/src/core/Directus/Container/Container.php(27): Pimple\\Container->offsetGet('logger')
#2 /var/www/XXX/directus/src/core/Directus/Application/CoreServicesProvider.php(236): Directus\\Container\\Container->get('logger')
#3 [internal function]: Directus\\Application\\CoreServicesProvider->Directus\\Application\\{closure}(Object(Directus\\Exception\\NotInstalledException))
#4 /var/www/XXX/directus/src/core/Directus/Hook/Emitter.php(291): call_user_func_array(Object(Closure), Array)
#5 /var/www/XXX/directus/src/core/Directus/Hook/Emitter.php(1 in /var/www/XXX/directus/src/core/Directus/Application/CoreServicesProvider.php on line 142, referer: https://XXX/admin/
```

## Other Context & Screenshots
I also tried configuring Directus with the provided scripts (as described [here](https://docs.directus.io/advanced/api/configuration.html#configure-with-script)) but this leads to the same issue. Login is also not possible with the configured credentials. The DB connection works fine as the user table was changed during script execution.

## Technical Details

- OS: Ubuntu 18.04
- Web Server: Apache 2.4.29
- PHP Version: 7.3.3
- Database: MySQL 5.7.27

**Comment by MyneLife** [comment]
Issue #2286 helped me to solve my issues. Why did not I find this issue earlier 😒 I did not meet two requirements:

1. The PHP module `curl` was missing. Install it using `apt-get install php-curl`.
2. I have `AllowOverride` enabled for all my virtual hosts, however my config was not sufficient for Directus. What I used so far in my `apache.conf`:
```
<Directory /var/www/>
        Options Indexes FollowSymLinks
        AllowOverride All
        Require all granted
</Directory>
```
What I added to my Directus virtual host config:
```
<Directory /var/www/XXX/directus>
    Options Indexes FollowSymLinks MultiViews
    AllowOverride All
    Order allow,deny
    allow from all
 </Directory>
```

There is a [Medium post](https://medium.com/directus/installing-directus-7-on-lamp-defc9c5b63e1) which is spot on. Maybe this helps someone else in future!
